src/metabase/api/automagic_dashboards.clj

```diff
@@ -47,7 +47,7 @@
     (tru "invalid value for rule name")))
 
 (def ^:private ^{:arglists '([s])} decode-base64-json
-  (comp json/decode codecs/bytes->str codec/base64-decode))
+  (comp #(json/decode % true) codecs/bytes->str codec/base64-decode))
```

Comment:
what's the `true` arg do? It might help readability by replacing that with a keyword like `:enable-whatever-im-enabling`. It's still logically true and helps code readability

Reply:
It keywordifies keys. And cheshire doesn't like anything except true for the value. :(

Reply:
I think this is a cheshire flag that enables/disables keyword keys

Reply:
I think elsewhere we're doing something like `(json/decode-string s keyword)`. I'm pretty sure one of the functions takes the function to apply to kws